### DISCUSSION TITLE
How to enabled `Protobuf` for kubernetes client

### DISCUSSION THREAD

**Discussion by zwangsheng** [fabric8io/kubernetes-client#5024] [Q&A] [answered]
Hi everyone,

I want to ask how to enabled `Protobuf` fpr kubernetes client, found example for go
```go
kubeConfig, err := clientcmd.BuildConfigFromFlags(s.Master, s.Kubeconfig)
if err != nil {
    return nil, err
}
kubeConfig.AcceptContentTypes = strings.Join([]string{runtime.ContentTypeProtobuf, runtime.ContentTypeJSON}, ",")
kubeConfig.ContentType = runtime.ContentTypeProtobuf
client, err := clientset.NewForConfig(restclient.AddUserAgent(kubeConfig, "content-type-example"))
```

But i can't find such like `withAcceptContentTypes` or `withContentType` function in kubernetes-client (java) config.

**Comment by shawkins** [comment, marked as answer]
> But i can't find such like withAcceptContentTypes or withContentType function in kubernetes-client (java) config.

Sorry, there is no support for Protobuf.
